ci/requirements-py35.yml

```diff
@@ -23,6 +23,7 @@ dependencies:
     - scipy
     - shapely  # pvfactors dependency
     - siphon  # conda-forge
+    - statsmodels
```

Comment:
You'll also need to add `statsmodels` to the optional dependencies in `setup.py`.